src/kanata/mod.rs

```diff
@@ -575,6 +583,11 @@ impl Kanata {
                 } else {
                     mmsh.distance = mmas.max_distance;
                 }
+                for &modifier in &self.move_mouse_speed_modifiers {
+                    mmsh.distance =
+                        (mmsh.distance as f32 * (modifier as f32 / 100 as f32)).round() as u16;
```

Comment:
I'm able to get 0 distance with low enough multiplier, I think we should set a minimum to 1